submissions/Lu-sure/index.js

```diff
@@ -0,0 +1,77 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Lu-sure/Lu-sure.github.io/tree/main/A_tiny_JS_world
+   Web app: https://lu-sure.github.io/A_tiny_JS_world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+const dog = {
+   species: 'dog',
+   name: 'Easy',
+   gender: 'female',
+   age: '2',
+   legs: 4,
+   hands: 0,
+   saying: {text: 'woof-woof!'},
```

Comment:
Why do we need an extra nested object here instead of just plain string?

Reply:
Obviously, something gone wrong during writing this . And the article about copying objects confused me. Fixed that already)